emergence_lib/src/graphics/lighting.rs

```diff
@@ -85,14 +102,17 @@ impl CelestialBody {
         CelestialBody {
             height: 2. * Height::MAX.into_world_pos(),
             hour_angle: 0.,
-            declination: 23.5 / 360. * PI / 2.,
+            declination: CelestialBody::DEFAULT_NOON_RADIANS * PI / 2.,
             travel_axis: PI / 6.,
             illuminance: 3e4,
             days_per_cycle: 29.53,
         }
     }
 }
 
+#[derive(Component, Debug)]
+pub(crate) struct Sun;
```

Comment:
Instead of "Sun", this could be something more specific to the fact this is the CelestialBody that drives the sky color.